app/src/main/java/com/grupo3/vinilos/artists/list/ArtistsListViewModel.kt

```diff
@@ -22,9 +22,10 @@ class ArtistsViewModel : ViewModel() {
                 val bands = repository.getBands();
                 val musicians = repository.getMusicians();
                 val artists = bands + musicians
+                val sortedArtists = artists.sortedBy { it.name }
```

Comment:
Necesitamos crear una variable extra ? 

Podemos solo hacer `artists = artists.sortrdBy(it.name)`

?

Reply:
Sí, lo acabo de cambiar por `var` en lugar de `Val`